core/src/main/resources/jenkins/monitor/EndOfLifeAdminMonitor/message.jelly

```diff
@@ -0,0 +1,45 @@
+<!--
+The MIT License
+
+Copyright (c) 2021 Tim Jacomb.
+Copyright (c) 2023 Mark Waite.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
+    <div class="alert alert-danger">
+        <form id="${it.id}" method="post" action="${rootURL}/${it.url}/act" name="${it.id}">
+            <f:submit name="yes" value="${%More Info}"/>
+            <l:isAdmin>
+                <f:submit value="${%Ignore}" name="no"/>
+            </l:isAdmin>
+        </form>
+        <h2>${it.displayName}</h2>
```

Comment:
might be worth making this smaller with e.g.
(not pulled locally currently)
```suggestion
        <h2 class="h3">${it.displayName}</h2>
```

Reply:
I like the smaller look that generates.  Added as 545315e523e013cff71f1b4c598b03d198e64b7d